Jenkinsfile

```diff
@@ -0,0 +1,497 @@
+)
+
+node('fedora-atomic') {
+    ansiColor('xterm') {
+        timestamps {
+            try {
+                deleteDir()
+                stage('ci-pipeline-rpmbuild-trigger') {
+                    def current_stage = "ci-pipeline-rpmbuild-trigger"
+
+                    echo "Current Stage is: ${current_stage}"
+
+                    // Python script to parse the ${CI_MESSAGE}
+                    writeFile file: "${env.WORKSPACE}/parse_fedmsg.py",
+                              text: "#!/bin/env python\n" +
+                                    "import json\n" +
+                                    "import sys\n\n" +
+                                    "reload(sys)\n" +
+                                    "sys.setdefaultencoding('utf-8')\n" +
+                                    "message = json.load(sys.stdin)\n" +
+                                    "if 'commit' in message:\n" +
+                                    "    msg = message['commit']\n\n" +
+                                    "    for key in msg:\n" +
+                                    "        print \"fed_%s=%s\" % (key, msg[key])\n"
+
+                    // Parse the ${CI_MESSAGE}
+                    sh '''
+                        #!/bin/bash
+                        set -xuo pipefail
+
+                        chmod +x ${WORKSPACE}/parse_fedmsg.py
+
+                        # Write fedmsg fields to a file to inject them
+                        if [ -n "${CI_MESSAGE}" ]; then
+                            echo ${CI_MESSAGE} | ${WORKSPACE}/parse_fedmsg.py > fedmsg_fields.txt
+                        fi
+                    '''
+
+                    // Load fedmsg fields as environment variables
+                    def fedmsg_fields = "${env.WORKSPACE}/fedmsg_fields.txt"
+                    def fedmsg_fields_groovy = "${env.WORKSPACE}/fedmsg_fields.groovy"
+                    convertProps(fedmsg_fields, fedmsg_fields_groovy)
+                    load(fedmsg_fields_groovy)
+
+                    // Check if package is in the package list for fedora-atomic host
+                    sh '''
+                        set +e
+                        branch=${fed_branch}
+                        if [ "${branch}" = "master" ]; then
+                          branch="rawhide"
+                          fed_branch="rawhide"
+                        fi
+                        echo "branch=${branch}" >> ${WORKSPACE}/job.properties
+                        echo "namespace=${namespace}" >> ${WORKSPACE}/job.properties
+                        
+                        # Verify this is a branch in our list of targets defined above in the parameters
+                        if [[ ! "${fed_branch}" =~ "${TARGETS}" ]]; then
+                            echo "${fed_branch} is not in the list"
+                            echo "topic=org.centos.prod.ci.pipeline.package.ignore" >> ${WORKSPACE}/job.properties
+                        elif                                           
```

Comment:
Does elif work with no check? Would this be else instead? It may very well work, I have just never seen elif with no boolean check